cmd/tclipd/static/css/base.css

```diff
@@ -1,5 +1,113 @@
+*, ::before, ::after {
```

Comment:
Lets split this commit into a separate PR

Reply:
Reverted this for now!